lib/elements/elements.dart

```diff
@@ -337,6 +338,59 @@ class DToast extends DElement {
   }
 }
 
+class SettingsModal extends DElement {
```

Comment:
I'm wondering if we could split this into a generic `ModalDialog` class and a task specific `SettingsModal` one? Or perhaps a `ModalDialog`class, w/o any knowledge of the keybindings info, and an instance of `ModalDialog` that has had some keybinding specific DOM placed in it.

Essentially, it'd be great to separate out the code that's generic to dialogs here (say we had more than one), and the code that's specific to the keybindings.